api/build.gradle

```diff
@@ -1,19 +1,30 @@
 plugins {
     id 'java'
+    id 'maven-publish'
 }
 
 sourceCompatibility = JavaVersion.VERSION_11
 targetCompatibility = JavaVersion.VERSION_11
 
 group 'by.borge.jarl'
 version '0.1.0-SNAPSHOT'
+description 'Jarl JVM API'
 
 repositories {
     mavenCentral()
+    maven { url 'https://repo.clojars.org/' }
+    maven {
+        url 'https://maven.pkg.github.com/fleskesvor/jarl'
+        credentials {
+            username = '_'
+            password = System.getenv("GITHUB_TOKEN")
+        }
+    }
+    mavenLocal()
 }
 
 dependencies {
-    implementation files("../core/target/jarl-${version}-standalone.jar")
+    implementation "by.borge.jarl:jarl:${version}"
```

Comment:
Because this change also published core as a Maven package, but if that doesn't make sense, this can be changed back, along with the added maven repositories above.